### PR TITLE
Enhancement USE constraints

### PR DESCRIPTION
**Pull request by stefaniapedrazzi**
Related to https://github.com/omichel/webots/pull/6228.

### CODE REVIEW

**Comment by stefaniapedrazzi** [comment]
:+1:

File: reference/def-and-use.md [hunk 1]
```diff
@@ -15,11 +15,27 @@ it in the ".wbt" or ".proto" file.
...
-can design a [PROTO](proto.md) model for this node and then add different PROTO
-instances to your world.
+### USE Exceptions
+
+Although it is permitted to name any node using the DEF keyword, there are some
+exceptions where USE nodes are not valid. A USE node is an exact copy of the
+corresponding DEF node, and thus no property of a USE node can change independently
+from the DEF node.
+
+For this reasons it is not allowed to insert USE nodes of [Solid](solid.md),
```
Comment (omichel) [inline reference/def-and-use.md]: reasons &rarr; reason

File: reference/track.md [hunk 2]
```diff
@@ -61,6 +61,9 @@ first child. Also note that the [Appearance](appearance.md) node of the
 [Shape](shape.md) and its `texture` and `textureTransform` fields have to be
 explicitly defined.
 
+Note that the [TextureTransform](texturetransform.md) node that is going to be automatically
+altered by the [Track](#track) functionality cannot be [USE](def-and-use.md) nodes.
```
Comment (omichel) [inline reference/track.md]: be USE nodes &rarr; be a USE node

**Comment by stefaniapedrazzi** [comment]
:+1:

**Pull request merged**